src/components/RuleCard/index.jsx

```diff
@@ -388,7 +388,9 @@ function RuleCard({ rule, ...props }) {
       <CardActions className={classes.cardActions}>
         <Button color="secondary">Duplicate</Button>
         <Button color="secondary">Update</Button>
-        <Button color="secondary">Delete</Button>
+        <Button color="secondary" onClick={handleRuleDelete}>
```

Comment:
`<Button color="secondary" onClick={() => onRuleDelete(rule)}>`. Let's also add `onRuleDelete` in the list of props: https://github.com/mozilla-frontend-infra/balrog-ui/blob/5d9cd9a6e3f1eb6fe406d7431c24fb3203641ab1/src/components/RuleCard/index.jsx#L399-L401

Perhaps something like:

```jsx
onRuleDelete: func.isRequired,
```